BeeKit/Model/DueBy/DueByTable.swift

```diff
@@ -0,0 +1,38 @@
+// Part of BeeSwift. Copyright Beeminder
+
+import SwiftyJSON
+
+@objc(DueByTable)
+public class DueByTable: NSObject, NSSecureCoding, Codable {
+    /// a dictionary of due by deltas and totals to daystamp strings
+    public private(set) var entries: [String: BeeminderDueByEntry]
+    
+    private enum Key: String {
+        case entries
+    }
+    
+    public init(entries: [String: BeeminderDueByEntry]) {
+        self.entries = entries
+    }
+    
+    public static var supportsSecureCoding: Bool { true }
```

Comment:
From: https://developer.apple.com/documentation/foundation/nssecurecoding

> An object that does override [init(coder:)](https://developer.apple.com/documentation/mapkit/mkmapitem/init(coder:)) must decode any enclosed objects using the [decodeObjectOfClass:forKey:](https://developer.apple.com/documentation/foundation/nscoder/1442558-decodeobjectofclass) method. For example:

It appears we aren't doing this, so shouldn't be returning true here?

Reply:
True. It seems I missed one entry.